serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -0,0 +1,30 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="upgrading-serverless"]
+= Upgrading {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+:context: upgrading-serverless
+
+toc::[]
+
+If you have previously installed a Technology Preview version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest version.
+
+In previous versions of {ServerlessProductName} (versions 1.5.0 and older), the only available *Subscription Update Channel* was `techpreview`.
+To upgrade to the latest version, you must update the channel to `preview-4.3`.
+
+.Prerequisites
+* You have installed a Technology Preview version of {ServerlessOperatorName}, and have selected Automatic updates during the installation process.
```

Comment:
@mgencur what if someone chose manual updates instead of automatic?

Reply:
The process differs after they switch the channel. For Manual updates they will have to approve the new install plan manually:

they will see "1 requires approval" in the Subscription detail window and open it
they will click "Preview Install Plan"
they will click "Approve"
at this point they're done and the upgrade happens

Reply:
I'm sorry. I had these comments stashed for about a week but didn't submit the review :(

Reply:
No worries @mgencur ! Thanks for your review.